### DISCUSSION TITLE
Is there a simple way to manage two serverpod projects on one computer?

### DISCUSSION THREAD

**Discussion by Frank1016** [serverpod/serverpod#2502] [Q&A] [answered]
I created one serverpod project, and everything is running fine with the database and serverAPI.  I stopped the container of this project with "docker compose stop". I created the second serverpod project, with no relation to the first project at all. 

After changing all default port numbers in development.yaml file, the docker containers can start running successfully, but the ports failed to open. Then I stopped the containers for this second project. Now the ports for the first project won't work either. 

No clue on what's wrong here, can anybody help?

update: after restarting VScode, the first project will work like normal, but the server for the second project still won't start. The following is my modified development.yaml:

![code](https://github.com/user-attachments/assets/a8670acd-faec-43d6-bf3d-d5c96e4a9ece)

**Comment by vlidholt** [comment, marked as answer]
You will need to update:

- Your `config/development.yaml` file to change the ports for `database` and `redis`. Here you also change the ports for the `apiServer`, `insightsServer`, and `webServer`.
- The `docker-compose.yaml` with the same ports for `postgres` and `redis`.
- Finally, make sure that you connect to the correct port from your Flutter app (when you create the client).
